tests/integration/typescript/test.ts

```diff
@@ -0,0 +1,96 @@
+
+const {
+  KEY_ID: keyId ='',
+  TOKEN_ID: tokenId ='',
+  PASSPHRASE: passphrase ='',
+  PRIVATE_KEY: privateKey ='',
+  PROJECT_ID: projectId ='',
+} = process.env
+
+// 🍌 init AuthProvider
+const authProvider = new AuthProvider({ keyId, tokenId, passphrase, privateKey, projectId })
+const apiKey = authProvider.fetchProjectScopedToken.bind(authProvider)
+
+async function testIam() {
+  const iamConfiguration = new IamConfiguration({ apiKey })
+  const api = new PoliciesApi(iamConfiguration)
+  const { data } = await api.getPolicies(tokenId, 'token')
+
+  return data
+}
+
+async function testVpa() {
+  const loginConfiguration = new LoginConfiguration({ apiKey })
+
+  const api = new ConfigurationApi(loginConfiguration)
+  const { data } = await api.listLoginConfigurations()
+
+  return data
+}
+
+async function testVer() {
+  const verifierConfiguration = new VpaConfiguration({ apiKey })
+
+  const api = new DefaultApi(verifierConfiguration)
+  const credentials = { "verifiableCredentials": [{"@context": ["https://schema.affinidi.com/TestEmployeeCredentialPersonNestedV1-2.jsonld"], "id": "claimId:6f4532f45933189d", "type": ["VerifiableCredential", "TestEmployeeCredentialPersonNested"], "holder": {"id": "did:elem:EiC9C2LaJrhrgrmXHyamViL15bcAT7P4wBfWefTfGfPoEA" }, "credentialSubject": {"data": {"name": "Jon Snow", "dateOfJoining": "2021-01-01", "otherDetails": {"githubLink": "https://github.com/jon-snow" } } }, "issuanceDate": "2021-10-21T20:58:21.217Z", "issuer": "did:elem:EiD78M2fskp7WgStA915l6xdynkm2PGIn4XtJLF3HFitLw;elem:initial-state=eyJwcm90ZWN0ZWQiOiJleUp2Y0dWeVlYUnBiMjRpT2lKamNtVmhkR1VpTENKcmFXUWlPaUlqY0hKcGJXRnllU0lzSW1Gc1p5STZJa1ZUTWpVMlN5SjkiLCJwYXlsb2FkIjoiZXlKQVkyOXVkR1Y0ZENJNkltaDBkSEJ6T2k4dmR6TnBaQzV2Y21jdmMyVmpkWEpwZEhrdmRqSWlMQ0p3ZFdKc2FXTkxaWGtpT2x0N0ltbGtJam9pSTNCeWFXMWhjbmtpTENKMWMyRm5aU0k2SW5OcFoyNXBibWNpTENKMGVYQmxJam9pVTJWamNESTFObXN4Vm1WeWFXWnBZMkYwYVc5dVMyVjVNakF4T0NJc0luQjFZbXhwWTB0bGVVaGxlQ0k2SWpBeU5tVmlaV0l3TTJFME56WmlPREJpWlRBd09EaGpaakUwTXpjMk9UTTVOR1l3TlRrNFpqZzNOVFEyT0dWbU1UUXdOakExTWpJMllURmxabUk1TkRaa01pSjlMSHNpYVdRaU9pSWpjbVZqYjNabGNua2lMQ0oxYzJGblpTSTZJbkpsWTI5MlpYSjVJaXdpZEhsd1pTSTZJbE5sWTNBeU5UWnJNVlpsY21sbWFXTmhkR2x2Ymt0bGVUSXdNVGdpTENKd2RXSnNhV05MWlhsSVpYZ2lPaUl3TXpKa1pESXlZekJqWTJZMFlUQmtOalV5TldFellUYzBaREUwT1dNeVl6TTNaalF6T1RrelpEUTVaV0ppTmpObU1qSmlObVUxWTJSa1lqSmhNREEwTURFaWZWMHNJbUYxZEdobGJuUnBZMkYwYVc5dUlqcGJJaU53Y21sdFlYSjVJbDBzSW1GemMyVnlkR2x2YmsxbGRHaHZaQ0k2V3lJamNISnBiV0Z5ZVNKZGZRIiwic2lnbmF0dXJlIjoiTHJoRDZhUzZMZHM4MnkwcWZmWjlSX1laUXhzSFNuQWxIWFpwUUVGaGoteEhsRExPb2pvM1piR2dBMDJ3b0hRVkdTUVhxdmtMTVVveTJjMmFjMWo3c1EifQ", "proof": {"type": "EcdsaSecp256k1Signature2019", "created": "2021-10-21T20:59:04Z", "verificationMethod": "did:elem:EiD78M2fskp7WgStA915l6xdynkm2PGIn4XtJLF3HFitLw#primary", "proofPurpose": "assertionMethod", "jws": "eyJhbGciOiJFUzI1NksiLCJiNjQiOmZhbHNlLCJjcml0IjpbImI2NCJdfQ..u6Cp6BLlSyVMVshDpiLzzcgeAAz7pSLJnFQaMSiBzlcbj93_zKZ3sermaHnlH_5qKwOsmMnRtx5VMDLqe8fSJw" } } ] }
```

Comment:
**issue (blocking):** please invlidate and create ci environment variables for secrets